src/components/EntryList/EntryList.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useEntries } from '../../context/EntriesContext';
+import './EntryList.css';
+
+export default function EntryList() {
+  const { entries } = useEntries();
+  return (
+    <div className="entrylist">
+      {entries.map(({ id, message, user }) => (
+        <div key={id} className="entry-border">
+          <div className="entry">
+            <h5>{message}</h5>
```

Comment:
For accessibility, [heading elements](https://webdesign.tutsplus.com/articles/the-importance-of-heading-levels-for-assistive-technology--cms-31753) should be used to describe the priority of the content, not the style, and should follow a logical sequence [without skipping levels](https://www.a11yproject.com/checklist/#dont-skip-heading-levels).